starport/services/chain/plugin-launchpad.go

```diff
@@ -215,18 +214,16 @@ func (p *launchpadPlugin) StartCommands(conf starportconf.Config) [][]step.Optio
 }
 
 func (p *launchpadPlugin) StoragePaths() []string {
+	home, _ := os.UserHomeDir()
 	return []string{
-		fmt.Sprintf(".%s", p.app.ND()),
-		fmt.Sprintf(".%s", p.app.NCLI()),
+		filepath.Join(home, "."+p.app.ND()),
```

Comment:
Concerning the ".", maybe we should create a specific function like `p.add.CONFIGDIR()` in case the standard name of the config dir change again

Reply:
moved to https://github.com/tendermint/starport/issues/495 :+1: